documentation/notifications.md

```diff
@@ -94,6 +95,12 @@ This is Stack Overflow's built in chat system. This is probably not useful for m
 
 Configure `bonfire_url` to be the full url including room and api key.
 
+### Others
+
+DNSControl can use many other notification methods via Shoutrrr, such as email (SMTP), Discord, Pushover and others. See the [Shoutrrr documentation](https://containrrr.dev/shoutrrr/v0.8/services/overview/) for a list of supported methods and configuration instructions.
```

Comment:
I would like to propose to change this version '0.8' to 'latest'. From the MkDocs generator a redirect takes place to the actual latest version.

```suggestion
DNSControl can use many other notification methods via Shoutrrr, such as email (SMTP), Discord, Pushover and others. See the [Shoutrrr documentation](https://containrrr.dev/shoutrrr/latest/services/overview/) for a list of supported methods and configuration instructions.
```